core/src/main/scala/scalaz/std/TypeConstraint.scala

```diff
@@ -7,9 +7,9 @@ trait TypeConstraintInstances {
     def compose[A, B, C](f: B <:< C, g: A <:< B) = f.asInstanceOf[A <:< C]
   }
 
-  val =:= = new Category[=:=] {
+  val === = new Category[===] {
     def id[A] = implicitly
-    def compose[A, B, C](f: B =:= C, g: A =:= B) = f.asInstanceOf[A =:= C]
+    def compose[A, B, C](f: B === C, g: A === B) = f.asInstanceOf[A === C]
   }
```

Comment:
Not sure whether this should be left alone or not...

Reply:
I think we want to leave this one alone

Reply:
also I don't think the cast looks safe.

Reply:
The cast is safe, but it is possible to do that with `===` (or `=:=` in 2.13) without it. I think you might want to add an instance for `===` if there is not one and keep the one for `=:=`, in case anyone already using it.